content-src/asrouter/templates/SimpleSnippet/SimpleSnippet.jsx

```diff
@@ -67,22 +68,60 @@ export class SimpleSnippet extends React.PureComponent {
       sendClick={props.sendClick} />);
   }
 
+  renderSectionHeader() {
+    const {props} = this;
+
+    // an icon and text must be specified to render the section header
+    if (props.content.section_title_icon && props.content.section_title_text) {
```

Comment:
Is there definitely never going to be a case of just the title being used?

Reply:
I'm pretty sure no, as the intent of this change is to match the icon/title style from the other sections. I'll confirm with Jean though.

Reply:
We covered this in the meeting, what you have implemented is correct